src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -766,6 +774,15 @@ export class HaAutomationEditor extends KeyboardShortcutMixin(LitElement) {
         mwc-fab.dirty {
           bottom: 0;
         }
+        .enabled_icon {
+          color: var(--primary-text-color);
+        }
+        .selected_menu_item {
+          color: var(--primary-color);
+        }
```

Comment:
This is not needed, I actually fixed this here <https://github.com/home-assistant/frontend/pull/7359/files#diff-57b9cba3e06172c4224cb56bc0ef55cebf53277a016de17e350271a75f22d5f3R77>

Reply:
Is that the correct link you wanted to paste? If yes, I must be missing the point how the quickbar is related to this PR.

Reply:
The `mdc-theme-text-secondary-on-background` variable is the one used for the icon, that is what you are fixing here right? (they both use `mwc-list-item`)

Reply:
Ah, I thought the link pointed to `ha-quick-bar.ts`. My change however, does one extra thing: The checkbox icon is colored in `primary-color`. Also, in case the menu option is disabled, what would happen with your change?

Reply:
I made a test by copying the mentioned style line from your PR. Does not work for me. All icons are black except the last one which I colored red.